deltachat-ios/Assets.xcassets/padlock.imageset/Contents.json

```diff
@@ -0,0 +1,23 @@
+{
+  "images" : [
+    {
+      "filename" : "padlock.svg",
+      "idiom" : "universal",
+      "scale" : "1x"
+    },
+    {
+      "filename" : "padlock.svg",
+      "idiom" : "universal",
+      "scale" : "2x"
+    },
+    {
+      "filename" : "padlock.svg",
+      "idiom" : "universal",
+      "scale" : "3x"
+    }
+  ],
+  "info" : {
+    "author" : "xcode",
+    "version" : 1
+  }
+}
```

Comment:
imagesets do not support svg on older system we're still supporting. so we need to use bitmaps with single, double and triple size (as we're doing for all the other images)

in fact, the old `ic_lock` already is in this format, however, probably not in the optimal resolution

Reply:
>  the old ic_lock already is in this format

yes, I noticed but it was always selecting the worst resolution when displaying. Do you know what is the minimum target for svg images icons?

Reply:
ios13 afaik